lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -497,8 +503,8 @@ def self.to_manifest_hash(iTrigger)
         'start_date'       => start_boundary ? start_boundary.strftime('%Y-%-m-%-d') : '',
         'start_time'       => start_boundary ? start_boundary.strftime('%H:%M') : '',
         'enabled'          => iTrigger.Enabled,
-        'minutes_interval' => Duration.to_minutes(iTrigger.Repetition.Interval) || 0,
-        'minutes_duration' => Duration.to_minutes(iTrigger.Repetition.Duration) || 0,
+        'minutes_interval' => iTrigger.Repetition.Interval == '' ? 0 : Duration.to_minutes(iTrigger.Repetition.Interval),
```

Comment:
I don't think these need to be changed... see https://gist.github.com/Iristyle/58ff446bd3f60e68d52df3868c917a0e#gistcomment-2387093 and `Duration.to_minutes` impl at https://github.com/puppetlabs/puppetlabs-scheduled_task/blob/master/lib/puppet_x/puppetlabs/scheduled_task/trigger.rb#L47-L55